docs/getting-started-guides/kubeadm.md

```diff
@@ -473,8 +473,10 @@ proposal](https://github.com/kubernetes/kubernetes/blob/master/docs/proposals/mu
 deb-packages are released for ARM and ARM 64-bit, but not RPMs (yet, reach out
 if there's interest).
 
-Currently, only the pod network flannel is working on multiple architectures.
-You can install it this way:
+Currently, only the pod networks flannel and Weave Net work on multiple architectures.
```

Comment:
Do you have a link for a guide for Flannel on arm?  Or are you saying to put that regardless, so they'll need to provide it?